tests/unit/multi.tcl

```diff
@@ -956,12 +956,6 @@ start_server {tags {"multi"}} {
         $rd close
     }
 
-    test "AUTH errored inside MULTI will add the reply" {
-        r config set requirepass ""
-        r multi
-        r auth no-user foobar
-        assert_error {WRONGPASS invalid username-password pair or user is disabled.} {r exec}
-    }
```

Comment:
what is this?

Reply:
No idea, probably bad merge, will revert.